### PR TITLE
#165273483 User should be able to receive notification

### PR DESCRIPTION
**Pull request by 3Nakajugo**
#### what does the pr do?
sends a notification to a user when a comment is made on an article they favorited

#### tasks to be completed.
- create app notifications
- create directory tests
- update URLs
- make migrations

#### how to tests PR
 - comment on a favorited article.
```
http://127.0.0.1:8000/api/articles/<airticle_id>/comments/
```
- get notifications
```
http://127.0.0.1:8000/api/notifications/
```
- get single notification
```
http://127.0.0.1:8000/api/notifications/<id>/
```
- enable and disable notifications
```
http://127.0.0.1:8000/api/notification/settings/
```
#### relevant PT story
[finishes#165273483](https://www.pivotaltracker.com/story/show/165273483)

### CODE REVIEW

File: authors/apps/notifications/models.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+from authors.apps.profiles.models import Follow
+
+
+class UserNotification(models.Model):
+    """Class for notifications."""
+
+    reciever = models.ForeignKey(User, to_field='username', on_delete=models.CASCADE)
+    message = models.CharField(max_length=255, null=True)
+    link = models.CharField(max_length=255, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
```
Comment (archibishop) [inline authors/apps/notifications/models.py:22]: @3Nakajugo Could we add an attribute to this model `read` which is initially a boolean `false`. Which can be updated to true when the user reads the notification.

Comment (3Nakajugo) [reply authors/apps/notifications/models.py:22]: okay. let me work on it

**Review by KabohaJeanMark — APPROVED** [review]
This LGTM. Good work Edna.

**Comment by KabohaJeanMark** [comment]
Please resolve the conflicts in the settings and urls.py files.

**Review by mwinel — CHANGES_REQUESTED** [review]
Ensure that the user receives the article link attached to the email.

**Review by mwinel — APPROVED** [review]

**Review by engjames — APPROVED** [review]

**Review by archibishop — APPROVED** [review]
LGTM @3Nakajugo I am merging this.

**Pull request merged**